src/shared/erros/dropdow-option-not-found.erros.js

```diff
@@ -0,0 +1,7 @@
+export default class DropdownOptionNotFound extends Error {
+    constructor(notFoundMessage) {

```

Comment:
You can just call this variable 'option'.

Reply:
Got it.